src/connection/unix-socket-connection.ts

```diff
@@ -1,7 +1,7 @@
 import * as net from 'net';
```

Comment:
Looks like this file doesn't do destruction-based imports either from earlier. Can you fix that up?

Reply:
Yeah I'll fix all of em